examples/defaulter-gen/_output_tests/marker/type.go

```diff
@@ -24,17 +24,32 @@ type Defaulted struct {
 	empty.TypeMeta
 
 	// +default="bar"
-	Field string `json:"Field,omitempty"`
-	// +default=0
-	OtherField int `json:"OtherField,omitempty"`
-
-	// Default is forced to 0
-	EmptyInt int
+	StringDefault string `json:"omitempty"`
 
 	// Default is forced to empty string
 	// Specifying the default is a no-op
 	// +default=""
-	EmptyString string
+	StringEmptyDefault string
```

Comment:
Can you add an example of a scalar pointer type?

Reply:
Added.